app/bundles/LeadBundle/Entity/LeadFieldRepository.php

```diff
@@ -158,6 +158,20 @@ public function compareValue($lead, $field, $value, $operatorExpr)
             } else {
                 return false;
             }
+        } elseif ($field === 'notifications') {
```

Comment:
I feel like the stuff which has its own bundle should not have conditions in another bundle. Do you think this could be moved to the NotificationBundle?